core/frontend/src/components/version-chooser/VersionCard.vue

```diff
@@ -116,10 +116,10 @@
             </v-btn>
             <v-spacer />
             <v-btn
-              color="warning"
+              color="success"
```

Comment:
I don't agree with success color for this button

Reply:
@joaoantoniocardoso check now